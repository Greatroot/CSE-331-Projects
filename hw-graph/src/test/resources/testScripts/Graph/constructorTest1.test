# Test to see if building an empty graph results in no nodes
# and no edges

CreateGraph emptyGraph


