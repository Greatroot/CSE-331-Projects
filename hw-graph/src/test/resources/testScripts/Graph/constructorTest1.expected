# Test to see if building an empty graph results in no nodes
# and no edges

created graph emptyGraph
emptyGraph contains: